d8error.py

```diff
@@ -80,9 +81,39 @@ def furtherTips(self):
     def print(self, i):
         display(Markdown)
     def title(self):
-        display(Markdown("## **Uh-o it seems we have an error!**"))
+        display(Markdown("## **There seems to be a <font color='red'>" + self.errorname+ "<font>**" + "."))
```

Comment:
In your PR description, can you provide a screenshot of the final result? My concern is that `color='red'` will be too glaring.